src/features/PostState/usePosts.js

```diff
@@ -1,16 +1,31 @@
-import { createContext, useContext, useEffect, useReducer } from 'react';
+import {
+    createContext,
+    useContext,
+    useEffect,
+    useReducer,
+    useRef,
+} from 'react';
 import postsReducer from './postsReducer';
 import airtableApi from 'features/api/airtable';
 
 export const PostsContext = createContext();
 
 export const PostsProvider = ({ children }) => {
-    const [{ posts, isLoading }, postsDispatcher] = useReducer(postsReducer, {
-        posts: [],
-        isLoading: false,
-        sortDirection: 'asc',
-        searchTerm: '',
-    });
+    const retrieveNextPage = useRef();
```

Comment:
The `useRef` hook lets us store the "state" of a scoped function, in this case the function returned by the airtable library that retrieves the next page.